docs/concepts/explore-available-balancer-pools/liquidity-bootstrapping-pool.md

```diff
@@ -2,11 +2,21 @@
 order: 3
 title: Liquidity Bootstrapping Pool
 ---
+
 # Liquidity Bootstrapping Pools (LBPs)
 
 ## Overview
 
-Liquidity Bootstrapping Pools (LBPs) are pools that can dynamically change token weighting (e.g 1/99 to 99/1 for TokenA/TokenB). LBPs use [Weighted Math](./weighted-pool/weighted-math.md) with time-dependent weights. The starting and end weights and times are selected by the pool owner, who also has the power to pause swaps. The pool owner is the only address that can join the pool.
+Liquidity Bootstrapping Pools (LBPs) are pools that can dynamically change token weighting (e.g 1/99 to 99/1 for TokenA/TokenB). LBPs use [Weighted Math](./weighted-pool/weighted-math.md) with time-dependent weights. The starting and end weights and times are selected by the pool owner. The pool owner is the only address that can join the pool and only remove liquidity after end time.
+
+::: info Create an LBP
+
+- [Create a LBP on Balancer Homepage](https://balancer.fi/lbp/create)
+- [Create a LBP with the balancer SDK](https://github.com/balancer/b-sdk/blob/main/examples/createAndInitPool/createAndInitLBPoolV3.ts)
+- [Create a LBP with a foundry script](https://github.com/balancer/balancer-v3-foundry-starter/pull/5/files)
+- Once the LBP has concluced, the token can be easily migrated to a standard pool on Balancer with this [tool]().
```

Comment:
```suggestion
- [Create an LBP on Balancer Homepage](https://balancer.fi/lbp/create)
- [Create an LBP with the balancer SDK](https://github.com/balancer/b-sdk/blob/main/examples/createAndInitPool/createAndInitLBPoolV3.ts)
- [Create an LBP with a foundry script](https://github.com/balancer/balancer-v3-foundry-starter/pull/5/files)
- Once the LBP has concluded, the token can be easily migrated to a standard pool on Balancer with this [tool]().
```
*Technically* it's correct to say "a liquidity bootstrapping pool", but that reads funny when you use the acronym.